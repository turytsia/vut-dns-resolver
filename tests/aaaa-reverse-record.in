-r -t -x -s dns.google 2001:4860:4860::8888