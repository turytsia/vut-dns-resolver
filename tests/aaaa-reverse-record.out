Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
 8.8.8.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.6.8.4.0.6.8.4.1.0.0.2.ip6.arpa., PTR, IN
Answer section (1)
 8.8.8.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.6.8.4.0.6.8.4.1.0.0.2.ip6.arpa., PTR, IN, 0, dns.google.
Authority section (0)
Additional section (0)
